vividus-to-zephyr-exporter/src/main/resources/org/vividus/zephyr/spring.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+       http://www.springframework.org/schema/beans https://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <bean id="zephyrExporterConfiguration" class="org.vividus.zephyr.exporter.ZephyrExporterConfiguration">
+        <constructor-arg index="0" value="${zephyr.project-key}"/>
+        <constructor-arg index="1" value="${zephyr.version-name}"/>
+        <constructor-arg index="2" value="${zephyr.cycle-name}"/>
+        <constructor-arg index="3" value="${zephyr.folder-name}"/>
+    </bean>
+
+    <bean class="org.vividus.zephyr.ZephyrFacade">
+        <constructor-arg index="0">
+            <bean class="org.vividus.jira.JiraFacade" />
```

Comment:
why do you need one more bean? such bean is already declared in jira-facade module